packages/bullet-list/README.md

```diff
@@ -0,0 +1,28 @@
+# hixme-ui bullet-list
+
+```bash
+npm i --save @hixme-ui/bullet-list
+```
+[View it live](https://hixme.github.io/hixme-ui/bullet-list)
```

Comment:
want to remove this until we have a hui-pages example of it?

Reply:
I'll make up a docs page for it real quick.